poly-synth/src/poly-synth.js

```diff
@@ -82,7 +82,7 @@ class PolySynth {
     for (let voice in this.sustainedVoices_) {
       this.sustainedVoices_[voice].lowPassFilter.frequency.value =
```

Comment:
Think I have mentioned this before; It's usually bad when the parent object accesses the children's specific member property (actually 3-level deep here) like this.